### PR TITLE
Fix #232: add --config path before `--` option/argument separator

### LINKED ISSUE #232: Compiling fails on latest update.
When running the following grunt task I get the error "You must compile individual stylesheets from the project directory."

compass:
      compile_all:
        options:
          sassDir: "sass"
          cssDir: ".tmp/css_staging"
          spriteLoadPath: "sprites"
          generatedImagesDir: ".tmp/sprites/images"
          specify: ["**/vendor.scss", "**/default.scss"]
          relativeAssets: false
          assetCacheBuster: false
          raw: "Sass::Script::Number.precision = 10\n"
          bundleExec: true
          sourcemap: true

This works fine in 1.0.1 but fails in 1.0.2.

### PR DESCRIPTION
**Pull request by phadej**
This should fix @mkilpatrick issue, he has `raw` in config.

Fixes #232

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -77,6 +77,15 @@ module.exports = function (grunt) {
           banner: '/* <%= pkg.name %> banner */'
         }
       },
+      issue232: {
+        options: {
+          raw: "Sass::Script::Number.precision = 10\n",
```
Comment (sindresorhus) [inline Gruntfile.js]: single-quotes

File: tasks/compass.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ module.exports = function (grunt) {
   var compass = require('./lib/compass').init(grunt);
 
   function compile(args, cb) {
+    grunt.log.verbose.writeln("Running command: " + args.join(' '));
```
Comment (sindresorhus) [inline tasks/compass.js]: single-quotes

and you can use verbose directly: `grunt.verbose.writeln`

File: tasks/compass.js [hunk 3]
```diff
@@ -65,7 +66,8 @@ module.exports = function (grunt) {
       }
 
       if (path) {
-        args.push('--config', path);
+        var doubleDashIdx = args.indexOf('--');
+        args.splice(doubleDashIdx === -1 ? args.length : doubleDashIdx, 0, '--config', path);
```
Comment (sindresorhus) [inline tasks/compass.js]: This could use a comment about why.

**Comment by sindresorhus** [comment]
Awesome. Thanks @phadej :)

Can you give this pull request a proper and descriptive title?

File: tasks/compass.js (hunk 2 above)
Comment (phadej) [inline tasks/compass.js]: You use `grunt.log.verbose.writeln` at https://github.com/gruntjs/grunt-contrib-compass/blob/a736fed14f0e419bd54408f5143686f12f6562cd/tasks/lib/compass.js#L108 and https://github.com/gruntjs/grunt-contrib-compass/blob/a736fed14f0e419bd54408f5143686f12f6562cd/tasks/lib/compass.js#L110. Should I change those too?

**Pull request closed without merging**